plugins/modules/content_view_version.py

```diff
@@ -204,6 +204,13 @@ def main():
 
     module.task_timeout = 60 * 60
 
+    if 'version' in module.foreman_params and '.' not in module.foreman_params['version']:
```

Comment:
What if you get "2.3.4.5"? Should we check for exactly one '.'?

Reply:
So something like:

```python
    if 'version' in module.foreman_params and module.foreman_params['version'].count('.') != 1:
        err_msg = "The 'version' needs to be in the format 'X.Y', not '{0}'".format(module.foreman_params['version'])
        if module.foreman_params['version'].count('.') > 1:
            module.fail_json(err_msg)
        else:
            try:
                major_version = int(module.foreman_params['version'])
                module.foreman_params['version'] = "{0}.0".format(major_version)
            except ValueError:
                module.fail_json(err_msg)
```

still doesn't catch "what.lol" as a version, but I'd not go that far right now?

Reply:
i thought checking a regex like `r"^(\d*)(?:\.(\d*))?$"`

Reply:
I've just used `^\d+\.\d+$` and then relied on `int()` throwing a `ValueError` ;)